debezium-core/src/test/java/io/debezium/converters/ByteBufferConverterTest.java

```diff
@@ -97,4 +101,35 @@ public void shouldConvertToConnectDataForNullValue() {
         assertThat(schemaAndValue.schema()).isEqualTo(Schema.OPTIONAL_BYTES_SCHEMA);
         assertThat(schemaAndValue.value()).isNull();
     }
+
+    @Test
+    public void shouldThrowWhenNoDelegateConverterConfigured() {
+        try {
+            converter.fromConnectData(TOPIC, Schema.OPTIONAL_STRING_SCHEMA, "Hello World");
+            fail("now expected exception thrown");
+        }
+        catch (Exception e) {
+            assertThat(e).isExactlyInstanceOf(DataException.class);
+        }
+    }
+
+    @Test
+    public void shouldConvertUsingDelegateConverter() {
```

Comment:
Could you add a test for specifying an option, e.g. include.schema set to false for the JSON converter?